src/nni_manager/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "nni",
-  "version": "NNI_VERSION",
+  "version": "1.0.0",
```

Comment:
If change this to 1.0.0,  we should remove following line from make file:
sed -ie 's/NNI_VERSION/$(NNI_VERSION)/' $(NNI_PKG_FOLDER)/package.json

Reply:
@goooxu